src/components/strategies/edit/overlapping/DepositOverlappingStrategy.tsx

```diff
@@ -152,9 +144,9 @@ export const DepositOverlappingStrategy: FC<Props> = (props) => {
           quote={quote}
           order0={order0}
           order1={order1}
-          marketPrice={externalMarketPrice}
+          marketPrice={oldMarketPrice?.toNumber()}
+          externalPrice={externalMarketPrice}
```

Comment:
Maybe you can use `useMarketPrice` in the component directly since it's a global variable and expect the current market price as props.